### ISSUE TITLE
cannot import complete database

### ISSUE THREAD

**Issue by jakob-r** [mboecker/lookup-server#74]
```
Importing dump into mysql
ERROR 1071 (42000) at line 12731: Specified key was too long; max key length is 767 bytes=================>                                            ] 64% ETA 0:29:59
2.30GiB 0:53:50 [ 746KiB/s] [=============================================================================>           
```

Using mariadb locally with a recent snapshot.

```
gunzip -c  ExpDB_SNAPSHOT.sql.gz | sed -n '12731,12732p;12732q'
```
```
CREATE TABLE `input` (
  `id` int(10) NOT NULL AUTO_INCREMENT,
```

**Comment by mboecker** [comment]
This is known to me. It's caused by the "generous" keys in the database. Until now I manually edited the .sql file to make this work, but I guess I should write a .patch or something for this.

**Comment by jakob-r** [comment]
Unfortunately I still get the error even with the pachted sql.gz file.

**Comment by jakob-r** [comment]
We could replace `varchar(512)` with `varchar(255)` and propably nothing would brake?

**Comment by mboecker** [comment]
> This fails:

Does this mean that this line fails with "blabla key size too large"? Or is it a different error?

**Comment by mboecker** [comment]
I didn't see you tried to fix it with varchar already. Did it work? My commit 00181ab58a925b41ce396d82df87947b390e422e should fix this. I tried the .sql file with CREATEs only, and there is no error on my system. Could you try the complete import one more time?

If it worked with your fix, please feel free to change it back and close this issue.

**Comment by mboecker** [comment]
The key error is fixed in 81f786b but I get the "table lock size exceeds..."-error again, but I think this will be fixed by editing /etc/mysql/my.cnf

**Comment by mboecker** [comment]
The preparation now works, if you edit your my.cnf file to contain "innodb_buffer_pool_size=256M" (other values might work).
